### PR TITLE
Incorrect usage of latest version rectified

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: docker-compose-tests.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
       RABBITMQ_USER: guest
       RABBITMQ_PASS: guest
   postgres:
-    image: registry.devshift.net/bayesian/coreapi-postgres
+    image: docker.io/samuzzal/coreapi-postgres
```
Comment (miteshvp) [inline docker-compose-tests.yml]: Let's use registry.centos.org one here

Comment (yzainee) [reply docker-compose-tests.yml]: surprising. it didnt pick the changes

Comment (yzainee) [reply docker-compose-tests.yml]: Please check now

Comment (yzainee) [reply docker-compose-tests.yml]: The changes show in the commit.. but in the file changes it shows different stuff.. dont know what this is

**Comment by miteshvp** [comment]
[test]

**Pull request merged**